operator/src/test/java/oracle/kubernetes/operator/helpers/Matchers.java

```diff
@@ -66,6 +78,39 @@ private static V1Container createContainer(String name, String image, String...
     return new V1Container().name(name).image(image).command(Arrays.asList(command));
   }
 
+  private static V1Container createInitContainer(String name, String image, String serverName, String... command) {
+    return new V1Container().name(name).image(image).command(Arrays.asList(command))
+            .env(PodHelperTestBase.getPredefinedEnvVariables(serverName));
+  }
+
+  private static V1Container createInitContainerWithEnvVar(String name, String image, String serverName,
+                                                           V1EnvVar envVar, String... command) {
+    List<V1EnvVar> envVars = new ArrayList<>(Arrays.asList(envVar));
+    PodHelperTestBase.getPredefinedEnvVariables(serverName).forEach(predefEnvVar ->
+            addIfMissing(envVars, predefEnvVar.getName(), predefEnvVar.getValue()));
+    return new V1Container().name(name).image(image).command(Arrays.asList(command))
+            .env(envVars);
+  }
+
+  protected static void addEnvVar(List<V1EnvVar> vars, String name, String value) {
+    vars.add(new V1EnvVar().name(name).value(value));
+  }
+
+  protected static boolean listHasEnvVar(List<V1EnvVar> vars, String name) {
+    for (V1EnvVar var : vars) {
+      if (name.equals(var.getName())) {
+        return true;
+      }
+    }
+    return false;
+  }
+
+  protected static void addIfMissing(List<V1EnvVar> vars, String name, String value) {
+    if (!listHasEnvVar(vars, name)) {
+      addEnvVar(vars, name, value);
+    }
+  }
+
```

Comment:
Those new methods (createXXX, listXXX and addXXX) do not seem to belong to this class.

Reply:
Thanks. These methods are used by the helper methods `hasInitContainer` and `hasInitContainerWithEnvVar` to verify if the init containers in the pod has appropriate pre-populated environment variables. Any suggestions on what would be a good place to move them?

Reply:
Now I see other helper methods in this class too.